### PR TITLE
Update UI components v2.0

### LINKED ISSUE #48: Update UI v.2.0
Remove some elements from footer and nav bar.
Change the favicon of the website.

### PR DESCRIPTION
**Pull request by HamzaMohibe**
Closed #48 
Remove some elements from the navbar and footer.
Change the icon of the website.
Update login and register page.
Add routes for each element

### CODE REVIEW

File: client/src/views/Calculate.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
   <header-nav-bar />
-  <div class="relative isolate overflow-hidden bg-gray-900 py-24 sm:py-32">
+  <div class="relative isolate overflow-hidden bg-gray-900 py-24 sm:py-15">
     <div
-      class="max-w-screen-xl mx-auto px-4 sm:px-6 lg:px-8 flex flex-col sm:flex-row justify-between items-center"
+      class="max-w-screen-xl px-4 sm:px-4 lg:px-4 flex flex-col sm:flex-row justify-between items-center"
     >
       <!-- Form -->
       <div class="w-full sm:w-1/2 rounded-lg">
```
Comment (tomstrong64) [inline client/src/views/Calculate.vue:8]: Seems to be an issue with alignment on larger screens, is this relating to the `sm:` part of the tailwind?
image

File: client/src/components/AppFooter.vue [hunk 2]
```diff
@@ -37,10 +37,10 @@
               <i class="fab fa-facebook-square"></i>
```
Comment (tomstrong64) [inline client/src/components/AppFooter.vue:37]: Icons not displaying properly on mine
image

File: client/src/views/auth/Login.vue [hunk 3]
```diff
@@ -7,30 +7,12 @@
...
-                Github
-              </button>
-              <button
-                class="bg-white active:bg-blueGray-50 text-blueGray-700 font-normal px-4 py-2 rounded outline-none focus:outline-none mr-1 mb-1 uppercase shadow hover:shadow-md inline-flex items-center font-bold text-xs ease-linear transition-all duration-150"
-                type="button"
-              >
-                <img alt="..." class="w-5 mr-1" :src="google" />
-                Google
-              </button>
+              <h6 class="text-blueGray-500 text-sm font-bold">Sign in</h6>
```
Comment (tomstrong64) [inline client/src/views/auth/Login.vue:10]: Issues with colours on login page
image

Unsure what colours register is supposed to be, but just for reference this is what it looks like for me
image

**Review by tomstrong64 — CHANGES_REQUESTED** [review]
Nice work, looks really good but there's a couple of things not displaying right on my system

**Review by tomstrong64 — APPROVED** [review]
LGTM, looks more consistent on my machine now, nice work

**Review by AhubSajid — APPROVED** [review]
Nice Work :)

**Pull request merged**